src/vaultwarden/models/permissive_model.py

```diff
@@ -0,0 +1,13 @@
+from pydantic import BaseModel
+
+from vaultwarden.utils.string_cases import pascal_case_to_camel_case
+
+
+class PermissiveBaseModel(
+    BaseModel,
+    extra="allow",
+    alias_generator=pascal_case_to_camel_case,
```

Comment:
Do you know when the alias_generator is called? Is it before Pydantic serialization/check?

In the case of an empty string, pascal[0].lower() will cause the program to crash.

Reply:
The alias generators are called ont each property of a Model and since properties can't be empty string, it should be OK.